docs/contributing.rst

```diff
@@ -24,24 +24,22 @@ We welcome contributions from external contributors, and this document describes
    versions <https://github.com/pyenv/pyenv-virtualenv>`_ or :code:`conda` to create virtual environments with `different Python
    versions <https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#managing-environments>`_.
 
-5. Install :code:`poetry` using the following command. :code:`poetry` is a better dependency resolver than :code:`pip`.
+5. Install :code:`poetry`. Instead of installing Poetry with :code:`pip`, follow the official Poetry installation guide using pipx:
+    Poetry Installation Guide <https://python-poetry.org/docs/#installation>_
 
-.. code-block:: bash
-
-    (local_venv) pip install poetry --upgrade pip
 
 6. Now, navigate to your local clone of the :code:`toqito` repository as shown below.
 
 .. code-block:: bash
 
-    (local_venv) cd toqito
+    cd toqito
```

Comment:
```suggestion
    $ cd toqito/
```

Reply:
Why do you need to specify `/`? 

I have often used `cd toqito` without any issues.

Reply:
Yes, you can do this, however the full path (if you say, use "tab" to complete the name) will add the `/` at the end. It also just makes it more explicit that it's a directory as well. So, yes, while it will work, I think the slash makes it more explicit.